auth_jwt/models/auth_jwt_validator.py

```diff
@@ -194,8 +194,8 @@ def _unregister_auth_method(self):
             try:
                 delattr(IrHttp.__class__, f"_auth_method_jwt_{rec.name}")
                 delattr(IrHttp.__class__, f"_auth_method_public_or_jwt_{rec.name}")
-            except AttributeError:
-                pass
+            except AttributeError as e:
+                _logger.info("AttributeError: %s", e)
```

Comment:
@yankinmax I would prefer to have these changes that do not relate to migration in a separate PR that we can backport easily.

Reply:
ok